shared/utils/ProsemirrorHelper.ts

```diff
@@ -176,8 +142,35 @@ export class ProsemirrorHelper {
    *
    * @returns True if the editor is empty
    */
-  static isEmpty(doc: Node) {
-    return !doc || doc.textContent.trim() === "";
+  static isEmpty(doc: Node, schema?: Schema) {
```

Comment:
Requires passing `schema` as an arg since the method is under `shared/` and schema is defined under `server/`.